src/server/es/index.js

```diff
@@ -53,14 +53,20 @@ class ES {
       },
     };
     validatedQueryBody.track_total_hits = true;
-    log.info('[ES.query] index, type, query body: ', esIndex, esType, JSON.stringify(validatedQueryBody));
+
+    var start = Date.now();
     return this.client.search({
       index: esIndex,
       body: validatedQueryBody,
     }).then((resp) => resp.body, (err) => {
       log.error(`[ES.query] error during querying: ${err.message}`);
       throw new Error(err.message);
-    });
+    }).finally(() => {
+       var end = Date.now();
+       var durationInMS = end - start;
+
+       log.info('[ES.query] DurationInMS:' + durationInMS + '. index, type, query body: ', esIndex, esType, JSON.stringify(validatedQueryBody));
+    })
```

Comment:
```suggestion
    });
```

Reply:
Geez. I've been working with too much Python. Thanks

How many engineers does it take to add a log :sweat_smile: